deps/src/main/java/com/microsoft/azure/sdk/iot/deps/twin/TwinMetadata.java

```diff
@@ -164,14 +188,43 @@ else if(key.equals(LAST_UPDATE_VERSION_TAG))
                 }
                 lastUpdatedVersion = ((Number)entry.getValue()).intValue();
             }
+            else if (key.equals(TwinMetadata.LAST_UPDATED_BY))
+            {
+                lastUpdatedBy = entry.getValue().toString();
+            }
+            else if (key.equals(TwinMetadata.LAST_UPDATED_BY_DIGEST))
+            {
+                lastUpdatedByDigest = entry.getValue().toString();
+            }
         }
+
         if((lastUpdatedVersion != null) || !Tools.isNullOrEmpty(lastUpdated))
         {
-            return new TwinMetadata(lastUpdated, lastUpdatedVersion);
+            return new TwinMetadata(lastUpdated, lastUpdatedVersion, lastUpdatedBy, lastUpdatedByDigest);
         }
         return null;
     }
 
+    /**
+     * Getter for the LastUpdatedBy.
+     *
+     * @return the {@code String} representing the configuration LastUpdatedBy.
+     */
+    public String getLastUpdatedBy()
+    {
+        return this.lastUpdatedBy;
+    }
+
+    /**
+     * Getter for the lastUpdatedByDigest.
```

Comment:
Line 209 uses PascalCase for the object, but this one uses camelCase

Reply:
will address in follow up PR.